nebula/ui/components/VPNControllerView.qml

```diff
@@ -148,6 +148,10 @@ Item {
                 target: animatedRings
                 visible: false
             }
+            PropertyChanges {
+                target: ipInfoPanel
+                isOpen: false
+            }
```

Comment:
Why do we only close the ip info panel on certain state changes instead of any state changes?

Reply:
For absolutely no good reason - thank you for the catch. Refactored for less verbosity. Also just realized this was missing a file and some other bits I probably lost jumping between branches. All here now.